tests/test_core.py

```diff
@@ -375,7 +377,7 @@ def test():
     printf("Alignment: %d addr: 0x%x\n", ((int)&v) % 16, (int)&v);
     printf("Alignment: %d addr: 0x%x\n", ((int)&m) % 16, (int)&m);
 }
-    ''', ('Alignment: 0 addr: 0xa20\nAlignment: 0 addr: 0xa60\n', 'Alignment: 0 addr: 0xe10\nAlignment: 0 addr: 0xe50\n')) # hardcoded addresses, for 2 common global_base values. this tracks if this ever changes by surprise. will need normal updates.
```

Comment:
I moved it up to the beginning of the inner function (just under `def test():`) so it's more visible. In my editor I couldn't see it without scrolling, and here in github I can't see it by default.